### PR TITLE
Mentor new

### PR DESCRIPTION
**Pull request by IdayatSanni**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md) 
- [x] I checked include the related website screenshots changes

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -801,4 +801,119 @@
...
+  type: long-term
+  index: 32
+  position: Java Software Engineer, Mettle
+  bio: Graduated in 2015 with BSc Computer Science, worked in gambling, fashion, SaaS. Switched into Fintech sector 2 months ago. Always coded in Java. Currently a Mentor for Riga Tech Girls (Jan - June 2023). Participated in several Hackathons pre-pandemic.
+  image: https://ca.slack-edge.com/TGYGA50DQ-U04PSFF0LMQ-5d4c714c1ab7-512
+  location: London, UK
+  languages: English, Russian
+  skills:
+    experience: 5-7 years
+    years: 5-7
```
Comment (nora-weisser) [inline _data/mentors.yml:907]:
```suggestion
    years: 7
```

File: _data/mentors.yml [hunk 2]
```diff
@@ -534,7 +534,7 @@
   index: 19
   position: Data Scientist, Keepflying
   bio: Competent Data Scientist capable of ensuring successful delivery of machine learning solutions and products for Multinational experience for the development and delivery of multi-tenant AI/ML solution. 5+ years research experience with 8+ publications, and time-critical completion of 4 machine learning projects in varies domain security, healthcare, machine translation and source code plagiarism detection.
-  image: https://media.licdn.com/dms/image/C4D03AQEeogRHkpw-cQ/profile-displayphoto-shrink_400_400/0/1646548153748?e=1681344000&v=beta&t=8qEpjvebzAQgKecsA_9m8Ek5wt_NKKdBpMiD5s6NrYE
+  image: https://media.licdn.com/dms/image/D4E03AQElKpYMThoSDA/profile-displayphoto-shrink_800_800/0/1676268606871?e=1681948800&v=beta&t=NKq8Lyk2xTM-BIXOHYvlzhwzG_g5tDwQYTmR-7Gx0yc
```
Comment (dricazenck) [inline _data/mentors.yml:537]: 👍

File: _data/mentors.yml [hunk 3]
```diff
@@ -801,4 +801,119 @@
...
+
+- name: Hersi Kopani
+  hours: 5
+  type: long-term
+  index: 30
+  position: Java Software Engineer, Volvo Group
+  bio: |
+    I am an experienced Java developer with industry experience building websites and web applications. Strong engineering background with a Master of Science (MSc) focused in Information Security from the University of Tirana. I'm passionate about learning the latest and greatest technologies in software development. I enjoy creating sleek and responsive applications in addition to being user-friendly. <br> I love working in fast-paced environments, having a team spirit, analyzing, planning, and solving various problems during the workday.
+    Interested in AI, ML, Information Security, IoT. Passionate programmer on java with its frameworks.
+  image: data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7
```
Comment (dricazenck) [inline _data/mentors.yml:845]: that is not an image link

Comment (IdayatSanni) [reply _data/mentors.yml:845]: Hi, I couldn't get an image link because her LinkedIn has no image. What should I replace it with?

Comment (dricazenck) [reply _data/mentors.yml:845]: take this one from his slack: https://ca.slack-edge.com/TGYGA50DQ-U04D90VJY66-84ca69c46836-512

Comment (dricazenck) [inline _data/mentors.yml:845]:
```suggestion
  image: https://ca.slack-edge.com/TGYGA50DQ-U04D90VJY66-84ca69c46836-512
```

Comment (dricazenck) [reply _data/mentors.yml]: I just pushed this change. Thanks for taking the time and updating.

**Review by dricazenck — APPROVED** [review]

**Pull request merged**